### PR TITLE
Initial support for conversion specifier words

### PR DESCRIPTION
**Pull request by pascal-fb-martin**
This is a first draft of the proposed change related to issue #299.
Tested by streaming a grey image with various "text_left" variants.

### CODE REVIEW

**Comment by tosiara** [comment]
Did not work for me:
```
[1:ml1] [NTC] [EVT] event_newfile: File of type 1 saved to: ./03-20170116114948-00-{host}.jpg
```

**Comment by tosiara** [comment]
Oh, sorry, made a mistake. Forgot to use `%` as prefix. Now `%{host}` is working as expected:
```
[1:ml1] [NTC] [EVT] event_newfile: File of type 1 saved to: ./02-20170116115104-03-vmware.jpg
```
We need update documentation and an example into motion-dist.config too

**Comment by tosiara** [comment]
Discussion on updating default config: https://github.com/Motion-Project/motion/issues/302
I'll do that if all agree

**Comment by pascal-fb-martin** [comment]
"update documentation": did you mean motion_guide.html? It is included.

motion-dist.config: humm. I noted that %$ (camera name) was not listed 
there.. Also, A you noted, it is in 3 places, each somewhat different 
(especially the last one). I also noticed conf.c, which seems like the 
point where the .conf template is located. Not sure what is the current 
policy, I followed the trail carefully and avoided too much innovation.

Thanks,
Pascal.

**Comment by tosiara** [comment]
Yeah, I meant the default config `motion-dist.conf.in`, as this is the primary place where I look for param description when configuring.

File: motion.c [hunk 1]
```diff
@@ -3214,6 +3214,62 @@ int myfclose(FILE* fh)
...
+ * fps     Equivalent to %fps.
+ */
+static void mystrftime_long (const struct context *cnt,
+                             int width, const char *word, int l, char *out)
+{
+#define SPECIFIERWORD(k) ((strlen(k)==l) && (!strncmp (k, word, l)))
+
+    if (SPECIFIERWORD("host")) {
+        char host[PATH_MAX];
+        gethostname (host, PATH_MAX);
```
Comment (tosiara) [inline motion.c]: I wonder if we should call gethostname only once at startup and save it as a string. So we don't need to call this function every single time. @jogu @Mr-Dave ?

**Pull request merged**